packages/core/types/index.js

```diff
@@ -1045,6 +1041,14 @@ export type Transformer<ConfigType> = {|
     options: PluginOptions,
     logger: PluginLogger,
   |}): Async<Array<TransformerResult | MutableAsset>>,
+  /** Do some processing after the transformation  */
```

Comment:
Can we mark this as `@experimental` since it still has some work left to be done on it? That should show up in the generated docs (though we'll have to double check).

Reply:
```suggestion
  /**
   * Do some processing after the transformation
   * @experimental
   */
```

will look like this after https://github.com/parcel-bundler/website/pull/1026:

![Bildschirmfoto 2022-05-08 um 18 21 02](https://user-images.githubusercontent.com/4586894/167305362-99907233-3b78-43b8-a207-d3fe0fae073a.png)

Reply:
@mischnic Thanks! I applied your commit suggestion! 👍